app/src/main/res/drawable/round_button.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item>
-        <shape android:shape="rectangle">
```

Comment:
Keep shape reactance as it was. Now add another rounded shape. Use this shapes background in fragment_donate.